src/core/mount.c

```diff
@@ -2063,7 +2063,9 @@ static void mount_enumerate(Manager *m) {
         mnt_init_debug(0);
```

Comment:
this  2nd commit should really be a separate PR. it should be fine to merge independently.